vc/pokecrystal11_vc.patch.template

```diff
@@ -0,0 +1,698 @@
+;00016c76h: E6 01 20 08 CD A0 6C CD 5A 04 18 E9 FA 63 CF F5 ; ? .蜖l蚙..辁c硝
+[print forbid 1]                                                   
+Mode = 1                                                                                                                  
+Address = {hex @ 4}
+Fixcode={patch}      
+
+
+                            
+[print forbid 2]                                                                           
+Mode = 6                                                                                       
+Type = 0                                                                                       
+Address = {hex @ 4}                                                                        
+MemAddress={hex hJoyPressed}                                                                           
+Fixcode={db NO_INPUT}                                                                                  
+ConditionType = 0                                                                              
+ConditionValueA = {dws_ wWindowStackPointer wWindowStackPointer+1 wMenuJoypad wMenuSelection wMenuSelection wMenuCursorY hJoyPressed hJoyPressed hJoyPressed hJoyPressed}                   
+ConditionValueB = {dws_ ==                  ==                    ==          >=             <=             ==           !=          !=          !=          !=         }                   
+ConditionValueC = {dws_ 0xdd                0xd3                  A_BUTTON    0x00           0x0f           0x03         D_DOWN      D_UP        B_BUTTON    NO_INPUT   }                   
+    
+; -----ddddfffffff99999ccccc77777----0xd9c7 no ..............Mem Write: pc32 = 0x230b addr = 0xd9c7 value = 0x8
+; 0xd9c7 is the room number.
+
+
+[print forbid 3]                                                          
+Mode = 6                                                                      
+Type = 0                                                                      
+Address = {hex @ 4}                                                           
+MemAddress={hex hJoyPressed}                                                            
+Fixcode={db NO_INPUT}                                                                 
+ConditionType = 0                                                             
+ConditionValueA = {dws_ wWindowStackPointer wWindowStackPointer+1 wMenuJoypad wMenuSelection wMenuCursorY wMapGroup         wMapNumber                wYCoord wXCoord hJoyPressed hJoyPressed hJoyPressed hJoyPressed}  
+ConditionValueB = {dws_ ==                  ==                    ==          ==             ==           ==                ==                        ==      ==      !=          !=          !=          !=         }  
+ConditionValueC = {dws_ 0xaf                0xdf                  NO_INPUT    0x00           0x01         MAPGROUP_CIANWOOD MAP_CIANWOOD_PHOTO_STUDIO 0x04    0x02    D_DOWN      D_UP        B_BUTTON    NO_INPUT   }  
```

Comment:
Same thing as above with the stack pointer and wMenuCursorY corresponding to the position of a menu option.